web-frame/src/app-feature/account-feature/pages/information-page/information-page.component.html

```diff
@@ -10,10 +10,19 @@ <h1 class="app-page__title">{{ getPageTitle() | translate }}</h1>
     {{ pageDetails.message | translate }}
   </p>
   <div class="app-form__actions">
-    <button class="app-button--with-icon" type="button" routerLink="/account/login">
-      <span class="text">
-        {{ 'ACCOUNT_FEATURE.INFORMATION_PAGE.GO_TO_LOGIN' | translate }}
-      </span>
-    </button>
+    <ng-container *ngIf="isAuthenticated; else anonymousAction">
+      <button class="app-button--with-icon" type="button" (click)="navigateToDestinationPage()">
+        <span class="text">
+          {{ 'GENERAL.BUTTON.GO_BACK' | translate }}
+        </span>
+      </button>
+    </ng-container>
+    <ng-template #anonymousAction>
```

Comment:
minor: Maybe we could rename to "annonymousUserActions"? I did not get what it was the intent from the first read.

Reply:
Done